test/helpers/oc/runner.go

```diff
@@ -134,3 +138,14 @@ func (r *runner) setArgs(args []string) {
 func (r *runner) setArgsStr(argstr string) {
 	r.args = sanitizeArgs(argstr)
 }
+
+func truncateString(str string, num int) string {
+	bnoden := str
```

Comment:
bnoden? What is the meaning of this variable?

Reply:
had copied it from go-playground, dont know what it means. will change.